### ISSUE TITLE
Missing Regions: eu-west-3; ap-northeast-3

### ISSUE THREAD

**Issue by agamzn** [awslabs/cfn-python-lint#461]
*cfn-lint version: (`cfn-lint --version`)*
`cfn-lint 0.9.0`
This is similar to  in that the regions are missing. These two regions show up on the official AWS [Documentation page here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html). 

```
cfn-python-lint/src/cfnlint/data/CloudSpecs on  master [?]
➜ diff region-list-final.txt specfile-list-final.txt
0a1
>
3d3
< ap-northeast-3
11d10
< eu-west-3
```

**Comment by agamzn** [comment]
PR is in https://github.com/awslabs/cfn-python-lint/pull/462

**Comment by kddejong** [comment]
Merged.  Will be released with #464.

Thanks again!
